extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/OpenTelemetryDestroyer.java

```diff
@@ -14,8 +16,8 @@ public void destroy(OpenTelemetry openTelemetry, CreationalContext<OpenTelemetry
             Map<String, Object> params) {
         if (openTelemetry instanceof OpenTelemetrySdk) {
             var openTelemetrySdk = ((OpenTelemetrySdk) openTelemetry);
-            openTelemetrySdk.getSdkTracerProvider().forceFlush();
-            openTelemetrySdk.getSdkTracerProvider().shutdown();
+            openTelemetrySdk.getSdkTracerProvider().forceFlush().join(1, SECONDS);
```

Comment:
Perhaps use some util for this + make it configurable?
... if / when other similar cases pop-up, which they probably will ...
Where util could also handle potential timeouts -- if / when they run over.

Reply:
I can try to use Config to retrieve a new property that sets that... 
Not sure about the util class because the reason for the `OpenTelemetryDestroyer.java` is just to call flush and shutdown.

Reply:
The util class will be better for testing.